src/main/java/net/smartcosmos/cluster/userdetails/DevKitUserDetailsService.java

```diff
@@ -32,4 +38,10 @@ PasswordEncoder passwordEncoder() {
 
         return new BCryptPasswordEncoder();
     }
+
```

Comment:
We have an evolving stylistic difference here.  I would prefer things like this be added in `@Configuration` classes unless there is a compelling reason to have them in the main class. We need to be consistent across applications about the location of things that are dynamic.  It's not likely but in the future we may need to make the encode configurable from the YML file, having it in a configuration class makes that easier.